bin/adam-shell

```diff
@@ -28,8 +26,34 @@ if [[ -z $@ && -n "$ADAM_OPTS" ]]; then
     echo "Run adam-shell instead as adam-shell <spark-args>"
 fi
 
+# Find ADAM cli assembly jar
+ADAM_CLI_JAR=
+if [ -f "$SCRIPT_DIR/repo" ]; then
```

Comment:
Why are we checking for a `/repo` directory here? Is this for backwards compatibility with appassembler?